packages/telemed-ehr/zambdas/src/get-telemed-appointments/helpers/fhir-utils.ts

```diff
@@ -43,10 +43,10 @@ export const getAllResourcesFromFhir = async (
         name: 'service-type',
         value: 'http://terminology.hl7.org/CodeSystem/service-type|telemedicine',
       },
-      {
-        name: 'date',
-        value: `le${searchDate.endOf('day')}`,
-      },
+      // {
```

Comment:
why is this not necessary?

Reply:
This filter is preventing us from seeing scheduled appointments